src/LibVLCSharp.Uno/Themes/PlaybackControls.xaml

```diff
@@ -353,7 +353,6 @@
                                                     VerticalAlignment="Stretch">
                                                 <ContentPresenter ContentTemplate="{TemplateBinding ContentTemplate}"
                                                                   ContentTransitions="{TemplateBinding ContentTransitions}"
-                                                                  Content="{TemplateBinding Content}"
```

Comment:
I believe this is correct that ContentPresenter should bind to the Content property by default so removing is not an issue.  Technically I think we can remove ContentTemplate  as well as this should be default.   If we used a different parameter base setting ContentSource allows renaming the binding on those two by default.

Reply:
As I understand this styles the volume flyout it seems to work with the change: 
![image](https://user-images.githubusercontent.com/1643324/122033369-4f513a80-cd85-11eb-8f3b-bfc8873b4b52.png)

Reply:
👍